test/test.js

```diff
@@ -292,4 +292,21 @@ describe('BigNumber.js', function () {
             BigNumber(5).plus(97).minus(53).plus(434).multiply(5435423).add(321453).multiply(21).div(2).val().should.equal("27569123001");
         });
     });
+
+    describe('very big numbers', function () {
+        it('should test main arithmetic on very large integers', function () {
```

Comment:
I could ... But the idea behind a separate describe was to single out errors that could be more due to large numbers rather than the logic of individual operations which,as you say, have their own tests. In other words, Any errors in this section would lead you to investigate the handling of large numbers rather than the logic of any specific method. Am away over w/e so will check in on Monday.

Regards, Dominic